README.md

```diff
@@ -20,3 +20,26 @@ Urban Suppiger <smiley@pfadiluzern.ch>
    login with user `ecamp2` and password `ecamp2`  
    import  `.database/ecamp_full_db.sql` into `ecamp2_dev` database
 5. Visit https://localhost:2000. Happy developing! :smile:
+
+# Install Guide (Webhosting, self-hosted)
+
+## Prerequisites
+- PHP
+- Mysql
+- composer with gd extension (on your machine)
```

Comment:
Not only on your machine. I think both GD and mbstring extension are needed on the webhosting.